learnAndCodeFinalProject/src/main/java/org/learnAndCode/Chef.java

```diff
@@ -0,0 +1,317 @@
+
+public class Chef extends User{
+
+    public Chef(String username, int roleId) {
+        super(username, roleId);
+    }
+
+    public static void displayMenu(PrintWriter writer, BufferedReader reader) throws IOException {
+        boolean continueSession = true;
+        while (continueSession) {
+            writer.println("Chef Menu:");
+            writer.println("1. Get recommendation");
+            writer.println("2. Roll out menu");
+            writer.println("3. Display voted items");
+            writer.println("4. Display menu items");
+            writer.println("5. Send Notification");
+            writer.println("6. View Feedback");
+            writer.println("7. Discard low rated Menu Items");
+            writer.println("8. View Discarded Menu Items");
+            writer.println("Please select an option:");
+
+            String option = reader.readLine();
+            System.out.println("Option selected: " + option);
+
+            if (option == null) {
+                writer.println("Invalid input. Please try again.");
+                continue;
+            }
+
+            switch (option) {
+                case "1":
+                    writer.println("Getting recommendation...");
+                    getRecommendation(writer);
+                    break;
+                case "2":
+                    writer.println("Rolling out menu...");
+                    rollOutMenu(writer, reader);
+                    break;
+                case "3":
+                    writer.println("Voted Items");
+                    displayVotedItems(writer);
+                    break;
+                case "4":
+                    writer.println("Displaying all menu items");
+                    displayMenuItems(writer);
+                    break;
+                case "5":
+                    sendNotification(writer, reader);
+                    break;
+                case "6":
+                    writer.println("Viewing feedback");
+                    viewFeedback(writer);
+                    break;
+                case "7":
+                    writer.println("Discarding low-rated items...");
+                    discardLowRatedItems(writer);
+                    break;
+                case "8":
+                    writer.println("Viewing discarded items...");
+                    viewDiscardedItems(writer, reader);
+                    break;
+                default:
+                    writer.println("Invalid option. Please try again.");
+                    continue;
+            }
+            continueSession = askToContinue(writer, reader);
+        }
+        writer.println("Thank you for using our cafeteria app!");
+    }
+
+    private static boolean askToContinue(PrintWriter writer, BufferedReader reader) throws IOException {
+        writer.println("Do you want to perform another function? (yes/no)");
+        String response = reader.readLine();
+        return "yes".equalsIgnoreCase(response);
```

Comment:
Don't hardcode values, instead store then in a variable with meaningful name and then use.